flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/AccessExecutionGraph.java

```diff
@@ -199,4 +207,12 @@ public interface AccessExecutionGraph extends JobStatusProvider {
      * @return The changelog storage name, or an empty Optional in the case of batch jobs
      */
     Optional<String> getChangelogStorageName();
+
+    /**
+     * Retrieves the count of pending operators waiting to be transferred to job vertices in the
+     * adaptive execution of batch jobs.
```

Comment:
Retrieves the count of pending operators waiting to be transferred to job vertices in the adaptive execution of batch jobs. This value will be zero if the job is submitted with a JobGraph or if it's a streaming job.

Reply:
Thanks for the input, revised.